doc/conf.py

```diff
@@ -29,7 +29,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest',
               'sphinx.ext.intersphinx', 'sphinx.ext.coverage',
-              'sphinx.ext.imgmath', 'sphinx.ext.viewcode']
+              'sphinx.ext.imgmath', 'sphinx.ext.viewcode', 'sphinxcontrib.bibtex']
```

Comment:
This is nice, but sphinxcontrib.bibtex is not installed on my machine which I guess is representative of other machines so you will need to add instructions in the getting going guide. Sorry. It is just sudo pip install sphinxcontrib-bibtex so it's not hard :-)

Reply:
Added.